message_center_compassion/model/contracts.py

```diff
@@ -140,7 +142,7 @@ def _invoice_paid(self, cr, uid, invoice, context=None):
                 # 1. Check if a GIFT was sent to GMC and prevent unrec
                 mess_ids = message_obj.search(cr, uid, [
                     ('invoice_line_id', '=', invoice_line.id),
-                    ('state', '=', 'success'),
+                    ('state', 'in', ['success', 'fondue']),

```

Comment:
fondue is a new state meaning "Fonds dû" => we must send the money of the gift to Compassion US.